app/models/requests/EmailRequest.scala

```diff
@@ -17,7 +17,8 @@
 package models.requests
 
 import models.EmailAddress
-import play.api.libs.json.{Json, OFormat}
+import play.api.libs.json.{JsValue, Json, OFormat, Writes}
```

Comment:
I notice that sometimes we're importing just .* and other times we making explicit references, whilst it doesn't impact the functionality that much. it might be worth considering one method or the other rather than a mix. I think logically it makes sense to the explicits as it'll reduce compile time a touch but this might not need addressing now in this ticket (or ever tbh)